.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: php
 
 php:
+  - 7.0
```

Comment:
That change is still unrelated so can't go in the same PR. Please ensure PRs are atomic.